custom/icds_reports/static/icds_reports/mobile/css/app.less

```diff
@@ -308,3 +308,13 @@
 .float-left {
   float: left;
 }
+.bar-chart {
+  .bg-white;
+  height: 415px;
+  margin-bottom: 5px;
+  overflow: auto;
+}
+svg {
```

Comment:
this selector feels a bit aggressive. what's it doing? any way to add a class or id attribute so it doesn't affect every svg image?

Reply:
I didnt mean this to be final pr. I will change it